### ISSUE TITLE
vitejs es6 type module

### ISSUE THREAD

**Issue by mtvbrianking** [brian2694/laravel-toastr#13]
Thanks for this amazing package.

I had some issue moving from webpack to vitejs

> Toastr.php

```diff
  public function message()
  {
      ...
-     $script = '<script type="text/javascript">';
+     $script = '<script type="module">';
      ...
```

Should we make this configurable?

**Comment by brian2694** [comment]
Hi, sorry for the late response, I hadn't seen the notification. Well, to be honest, I don't have any experience with Vite, so I need your help.

If I add a kind of switch to change the script's 'type' between 'text/javascript' and 'module', would it make it work correctly? Do I need anything else?

**Comment by brian2694** [comment]
I did some testing with Laravel 10, Vite, and the Vue installation used with Breeze.

I added this to the script in Welcome.vue:

`onMounted(() => {
console.log("Llegué al onMounted ubicado en Welcome.vue");
toastr.success("I am Welcome.vue");
});`

In this case, it works perfectly. I'll attach some screenshots for you.

I would like to replicate your scenario, so I need more information.

![Toastr Welcome](https://github.com/brian2694/laravel-toastr/assets/5945156/a13cb1b6-9ce9-494d-8412-3e0c69142699)
![Toastr dashboard](https://github.com/brian2694/laravel-toastr/assets/5945156/0995e655-2dd9-4edc-bd7e-c610d536bcd0)
